stanovy_vpsFree.cz.md

```diff
@@ -154,149 +166,200 @@ informačním systému spolku
+členů Rady a Kontrolní komise; volební řád může být schválen společně s
+programem schůze.
 
 ### RADA
 
 ### §15 Postavení a působnost
 
-1. Rada je statutárním orgánem spolku, který řídí činnost Spolku
-2. Rada rozhoduje o všech činnostech spolku, které nejsou stanovami vyhrazeny do
-působnosti Schůze členů nebo Kontrolní komise. Při své činnosti se Rada řídí
-stanovami Spolku a ustanoveními Schůze členů.
-3. Rada:
+1. Rada je kolektivním statutárním orgánem spolku, který řídí činnost Spolku
+2. Za Radu jedná navenek její předseda; v jeho nepřítomnosti jiný pověřený člen
+Rady 
+3. Rada rozhoduje o všech činnostech spolku, které nejsou stanovami vyhrazeny
+do působnosti Schůze členů nebo Kontrolní komise. Při své činnosti se Rada řídí
+stanovami Spolku a ustanoveními Schůze členů
+4. Rada:
   1. dbá na plnění cílů Spolku,
-  2. zajišťuje řádné plnění zákonných povinností Spolku
+  2. zajišťuje řádné plnění zákonných povinností Spolku,
   3. volí Hospodáře, který spravuje majetek Spolku a zajišťuje řádné vedení
-účetnictví, statistik a jiných záznamů podle platných právních předpisů,
+  účetnictví, statistik a jiných záznamů podle platných právních předpisů,
   4. volí Předsedu, který jedná jménem Spolku,
   5. spravuje informační systém,
   6. rozhoduje o přijetí člena,
   7. rozhoduje o vyloučení člena,
   8. vede seznam členů,
   9. poskytuje informace členům o své činnosti prostřednictvím informačního
-systému spolku
-  10. svolává jednání Schůze členů i mimořádné jednání Schůze členů
-4. Rada zajistí Kontrolní komisi přístup k veškerým dokumentům související s
+  systému spolku
+  10. svolává řádné jednání Schůze členů i mimořádné jednání Schůze členů
+5. Rada zajistí Kontrolní komisi přístup k veškerým dokumentům související s
 činností spolku
-5. Rada je kdykoliv povinna podat vysvětlení jakéhokoliv rozhodnutí Kontrolní
+6. Rada je kdykoliv povinna podat vysvětlení jakéhokoliv rozhodnutí Kontrolní
 komisi
-6. Rada má povinnost předložit Kontrolní komisi ke schválení veškeré plánované
+7. Rada má povinnost předložit Kontrolní komisi ke schválení veškeré plánované
 uzavření obchodů s členem Spolku nebo s osobami jemu blízkými v pracovněprávním
 nebo jiném obdobném vztahu.
 
 ### §16 Složení a funkční období
 
 1. Rada má, pokud to členská základna dovoluje, minimálně 3 členy a maximálně
-pět členů.
+pět členů
 2. Členy Rady volí Schůze členů
 3. Členem Rady může být pouze fyzická osoba, která:
   1. dosáhla věku 18 let a je způsobilá k právním úkonům,
   2. je členem Spolku
   3. je bezúhonná ve smyslu zákona o živnostenském podnikání
 4. Členství v Radě je neslučitelné s členstvím v Kontrolní komisi
-5. Funkční období člena Rady není omezeno časově. Člen Rady může být odvolán
+5. Funkční období člena Rady není omezeno časově; člen Rady může být odvolán
 pouze Schůzí členů na jejím pravidelném nebo mimořádném zasedání. Výkon funkce
-člena Rady skončí dnem zvolením nového člena.
+člena Rady skončí dnem zvolením nového člena
```

Comment:
Vím že to je jen syntaktická úprava, ale mám připomínku k originálnímu znění:
členů rady může být 3-5, hypotetická situace - máme 5 členů, schůze 2 členy odvolá, nikoho místo nich nezvolí. Podle mě by měli oba skončit ihned, Rada bude stále v souladu se stanovami. Pravděpodobně je odvoláme, protože je tam nechceme z nějakého závažného důvodu, nebo protože o to z nějakého závažného důvodu daný člen rady požádá. Tudíž by odvolaný člen rady měl skončit ihned. Proč by měli zůstat v radě dokud nezvolíme někoho jiného? To by mělo smysl pouze pokud by počet členů rady klesů pod 3.

Reply:
OK. Jen mi to přišlo zbytečně složité.